metricbeat/module/vsphere/host/host.go

```diff
@@ -93,6 +93,12 @@ func (m *HostMetricSet) Fetch(ctx context.Context, reporter mb.ReporterV2) error
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+	period := m.Module().Config().Period
+	if !isValidPeriod(period.Seconds()) {
+		m.Logger().Errorf("Invalid period: %v", period)
```

Comment:
Be consistent with the error logging. The same function call above has a different log message.

Reply:
Sure